### PR TITLE
LPS-147499 | Translation Manager default language is available with no other languages options

### PR DESCRIPTION
**Pull request by Hijky**
**Background**
Create automation tests for Translation Manager

**Test Plan**

Create a page with the widget JS Components Sample
Click and view select translation language menu
Assert the default language is available and no other languages options
Click Manage Translations
Assert the link to Manage Translations is available

**JIRA Ticket:**
https://issues.liferay.com/browse/LPS-147499

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path [hunk 1]
```diff
@@ -785,6 +785,16 @@
...
+<tr>
+	<td>SELECT_TRANSLATION_LANGUAGE</td>
+	<td>//div[contains(@class,'col') and h3='${key_title}']//button[contains(@class,'dropdown-toggle')]//*[name()='svg'][contains(@class,'lexicon-icon-${key_currentLocale}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>MANAGE_TRANSLATIONS</td>
+	<td>//button[contains(@class,'dropdown-item')]//div[contains(@class,'autofit-section')and text()='Manage Translations']</td>
+	<td></td>
+</tr>
```
[the remark this answers is not in the document]
Comment (john-co) [reply portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path:797]: Or maybe can just use the existing path `Button#TRANSLATIONS_LOCALIZATION`?

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path [hunk 2]
```diff
@@ -785,6 +785,16 @@
 	<td>//a[contains(@class,'btn-sm btn-primary') and text()='Done']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>SELECT_TRANSLATION_LANGUAGE</td>
+	<td>//div[contains(@class,'col') and h3='${key_title}']//button[contains(@class,'dropdown-toggle')]//*[name()='svg'][contains(@class,'lexicon-icon-${key_currentLocale}')]</td>
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path:790]: Try to use elements or attributes that are more related to the name of the path. ie: class = "translation-manager" or "translation-language".

I don't have the sample portlet available to view right now but something like `//*[contains(@class,'select-translation-language')]//button[contains(@class,'dropdown-toggle')]`

If `select-translation-language` doesn't exist then we can rename the path name to something that's more closely associated to the available html attributes.

File: modules/apps/translation/translation-test/src/testFunctional/tests/TranslationsManager.testcase [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+			AssertElementNotPresent(
+				key_locale = "zh-CN",
+				locator1 = "Translation#DROPDOWN_MENU_ITEM");
+
+			AssertElementNotPresent(
+				key_locale = "es-ES",
+				locator1 = "Translation#DROPDOWN_MENU_ITEM");
+
+			AssertElementPresent(locator1 = "Button#MANAGE_TRANSLATIONS");
+		}
```
[the remark this answers is not in the document]
Comment (Hijky) [reply modules/apps/translation/translation-test/src/testFunctional/tests/TranslationsManager.testcase:74]: I have added two language options that do not appear in assertnotpresent in my pull

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path (hunk 1 above)
[the remark this answers is not in the document]
Comment (Hijky) [reply portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path:797]: I will add a new path named TranslationManager

File: modules/apps/translation/translation-test/src/testFunctional/macros/Translations.macro [hunk 4]
```diff
@@ -742,6 +742,15 @@ definition {
 			locator1 = "ContentRow#ENTRY_CONTENT_ENTRY_NAME");
 	}
 
+	macro viewSelectTranslationLanguageMenu {
```
[the remark this answers is not in the document]
Comment (Hijky) [reply modules/apps/translation/translation-test/src/testFunctional/macros/Translations.macro:745]: I will add a new macro named TranslationManager and rename this macro in it

**Pull request closed without merging**